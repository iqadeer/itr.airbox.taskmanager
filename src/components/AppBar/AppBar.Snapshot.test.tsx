import React from 'react';
import renderer from 'react-test-renderer';
import ApdAppBar from './AppBar';
import { MemoryRouter as Router } from 'react-router-dom';

it('should have correct links', () => {
  const appBar = renderer.create(
    <Router>
      <ApdAppBar></ApdAppBar>
    </Router>
  );
  expect(appBar).toMatchSnapshot();
});
