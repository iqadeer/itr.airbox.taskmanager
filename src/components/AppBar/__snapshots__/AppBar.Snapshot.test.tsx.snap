// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should have correct links 1`] = `
<div
  className="makeStyles-root-1"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h6
        className="MuiTypography-root makeStyles-title-5 MuiTypography-h6 MuiTypography-noWrap"
      >
        <a
          aria-current="page"
          className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-link-2 MuiTypography-colorPrimary makeStyles-activeLink-3"
          href="/"
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          style={Object {}}
        >
          Airbox Task Manager
        </a>
      </h6>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        <a
          aria-current={null}
          className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-link-2 MuiTypography-colorPrimary"
          href="/multiple-tasks"
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
        >
          Manage multiple tasks
        </a>
        <a
          aria-current={null}
          className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-link-2 MuiTypography-colorPrimary"
          href="/organisation-tasks"
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
        >
          Manage organisation tasks
        </a>
        <a
          aria-current={null}
          className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-link-2 MuiTypography-colorPrimary"
          href="/individual-tasks"
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
        >
          Manage individual tasks
        </a>
      </p>
    </div>
  </header>
</div>
`;
